### PR TITLE
Update xblockmufi installation in requirements files

### PR DESCRIPTION
**Pull request by tkim135**
@stvstnfrd These are the installation changes you wanted to see in the requirements/edx/stanford.txt, CMS, and LMS files.

CC @saharm1

### CODE REVIEW

File: requirements/edx/stanford.txt [hunk 1]
```diff
@@ -2,11 +2,11 @@ unidecode
 xblock-image-modal==0.4.1
 -e git+https://github.com/openlearninginitiative/xblock-image-coding.git@cee23e3d59943066159ca13839c82a12d9ccf749#egg=xblock_image_coding
 -e git+https://github.com/Stanford-Online/xblock-qualtrics-survey@release/v0.1.2#egg=xblock-qualtrics-survey
--e git+https://github.com/Stanford-Online/xblock-free-text-response@release/v0.1.9#egg=xblock-free-text-response
+-e /edx/app/edxapp/xblocks/xblock-free-text-response
 -e git+https://github.com/Stanford-Online/xblock-grademe.git@v0.1.1#egg=grademebutton
 -e git+https://github.com/edx-solutions/xblock-ooyala.git@32d52edaa820dbdbf846d8a84f6bccbf0b9c8218#egg=xblock_ooyala_player-master
 -e git+https://github.com/Stanford-Online/xblock-submit-and-compare.git@release/0.5.2#egg=xblock-submit-and-compare
--e git+https://github.com/Stanford-Online/xblock-mufi.git@ee853b8a7668a87c27d13d55c0d149a04b8657b8#egg=xblock_mufi-master
+-e /edx/app/edxapp/xblocks/xblock-mufi
```
Comment (caesar2164) [inline requirements/edx/stanford.txt]: We definitely don't want this.

File: requirements/edx/stanford.txt [hunk 2]
```diff
@@ -2,11 +2,11 @@ unidecode
 xblock-image-modal==0.4.1
 -e git+https://github.com/openlearninginitiative/xblock-image-coding.git@cee23e3d59943066159ca13839c82a12d9ccf749#egg=xblock_image_coding
 -e git+https://github.com/Stanford-Online/xblock-qualtrics-survey@release/v0.1.2#egg=xblock-qualtrics-survey
--e git+https://github.com/Stanford-Online/xblock-free-text-response@release/v0.1.9#egg=xblock-free-text-response
+-e /edx/app/edxapp/xblocks/xblock-free-text-response
```
Comment (caesar2164) [inline requirements/edx/stanford.txt]: We don't want this.

File: requirements/edx/stanford.txt (hunk 1 above)
Comment (tkim135) [reply requirements/edx/stanford.txt]: @caesar2164 This PR was just for Steven to know what was in my requirements file while I was debugging the XBlock

Comment (tkim135) [reply requirements/edx/stanford.txt]: I don't think we were planning on merging this

**Comment by tkim135** [comment]
@stvstnfrd i just added a commit that upgrades the version in stanford.txt

**Pull request merged**